include/ultra64/xstdio.h

```diff
@@ -32,7 +32,7 @@ typedef struct {
 typedef void* (*PrintCallback)(void*, const char*, size_t);
 
 int _Printf(PrintCallback, void* arg, const char* fmt, va_list ap);
-void _Litob(_Pft* args, char code);
+void _Litob(_Pft* args, char type);
```

Comment:
this change shouldn't be there once you merge main (or run formatter) then we're good to go